.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+name: figures tests
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python: 2.7
+            tox-env: py27-ginkgo
+          - python: 2.7
+            tox-env: py27-hawthorn
+          - python: 2.7
+            tox-env: py27-hawthorn_multisite
+          - python: 3.5
+            tox-env: py35-juniper
+          - python: 3.8
+            tox-env: py38-juniper
+          - python: 3.8
+            tox-env: lint
+          - python: 3.8
+            tox-env: edx_lint_check
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: install dependencies
+        run: |
+          pip install tox flake8
+      - name: Uses Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install -g yarn
+      - run: |
+          cd frontend && yarn install && cd ..
+      - name: flake8
```

Comment:
I copied this step over from the Travis config, but it seems like it's a bit redundant since the `lint` tox testenv runs `flake8` anyway. We probably don't need to do it separately on each.